src/modules/account/components/augur-status/activity.tsx

```diff
@@ -20,8 +22,15 @@ export default class Activity extends React.Component<
     selected: constants.TIMEFRAME_OPTIONS[0].id
   };
 
+  componentDidMount() {
+    this.props.updatePlatformTimeframeData({
+      endTime: constants.TIMEFRAME_OPTIONS[0].id
```

Comment:
need to calculate endTime to send to augur-node.  I'll move how PnL chart does it to a helper